### DISCUSSION TITLE
animate.py not working

### DISCUSSION THREAD

**Discussion by iamfarmaan** [highfestiva/finplot#414] [Q&A]
I tried to run the animate.py on spyder IDE, but i'am getting the following error:

 spot_plot,label_plot = fplt.live(2)
Traceback (most recent call last):

  File "C:\Users\farmaan\AppData\Local\Temp\ipykernel_13356\629574286.py", line 1, in <module>
    spot_plot,label_plot = fplt.live(2)

AttributeError: module 'finplot' has no attribute 'live'

How do I resolve this?

**Comment by highfestiva** [comment]
You need to either use an older test (v1.9.0), or patch your finplot version to head of master. `live()` was introduced in f31a4475, whereas the latest released finplot is v1.9.0=32beac74. GL!
